app/models/task.rb

```diff
@@ -20,6 +20,7 @@
 
 class Task < ApplicationRecord
   has_and_belongs_to_many :campaigns
+  has_and_belongs_to_many :events
```

Comment:
I know the feeling, I do the same with... pretty much everything :P But I do feel like these discussions are useful, if not for now, for the future